### PR TITLE
[pdata] export NewSliceFromRaw

### PR DESCRIPTION
**Pull request by TylerHelmuth**
**Description:** 
Exports `internal.NewSliceFromRaw` to make it easier to make a slice from know values.

**Link to tracking Issue:** 
#5617

**Testing:**
Ran existing tests

### CODE REVIEW

File: pdata/pcommon/alias.go [hunk 1]
```diff
@@ -93,3 +93,8 @@ var (
 	// NewMapFromRaw creates a Map with values from the given map[string]interface{}.
 	NewMapFromRaw = internal.NewMapFromRaw
 )
+
+var (
+	// NewSliceFromRaw creates a Slice with values from the given []interface{}.
+	NewSliceFromRaw = internal.NewSliceFromRaw
+)
```
Comment (dmitryax) [inline pdata/pcommon/alias.go:100]: Doesn't seem like var block is useful here
```suggestion
// NewSliceFromRaw creates a Slice with values from the given []interface{}.
var NewSliceFromRaw = internal.NewSliceFromRaw
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 ### 💡 Enhancements 💡
 
 - Add `linux-ppc64le` architecture to cross build tests in CI
+- Expose `internal.NewSliceFromRaw` as `NewSliceFromRaw` in pdata (#5679)
```
Comment (dmitryax) [inline CHANGELOG.md:8]:
```suggestion
- Expose `pcommon.NewSliceFromRaw` function (#5679)
```

**Review by dmitryax — APPROVED** [review]

**Comment by dmitryax** [comment]
@codeboten I believe this is ready for merge

**Review by codeboten — APPROVED** [review]

**Pull request merged**